### PR TITLE
Fixed the issue , Please Review ! 

### PR DESCRIPTION
**Pull request by Raghav-byte**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by Raghav-byte** [comment]
Thanks !!

**Comment by mitulsavani** [comment]
> Looks good to me

@kaiwalyakoparkar Not really !

File: _data/Raghav-byte/Raghav-byte.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Raghav Shukla
+institution: Shri Shankracharya Institute of Prof. Management and Tech.
+profile_pic: Raghav-byte.jpg
+quote: "Why Fall In Love When You Can Fall Asleep"
```
Comment (mitulsavani) [inline _data/Raghav-byte/Raghav-byte.md:4]: Please follow the given instruction by removing the quotes to maintain the same format

**Comment by Raghav-byte** [comment]
> > Looks good to me
> 
> @kaiwalyakoparkar Not really !

Why ?👀

File: _data/Raghav-byte/Raghav-byte.md (hunk 1 above)
Comment (Raghav-byte) [reply _data/Raghav-byte/Raghav-byte.md:5]: i did the needful , please check again !
Thanks

Comment (mitulsavani) [reply _data/Raghav-byte/Raghav-byte.md:5]: Seems like you haven't removed the quotes yet

Comment (mitulsavani) [reply _data/Raghav-byte/Raghav-byte.md:5]: > Seems like you haven't removed the quotes yet

Let me know when you resolve this

**Comment by mitulsavani** [comment]
Let me know when you are done working on [this](https://github.com/education/graduation/pull/4083#discussion_r430247467)

File: _data/Raghav-byte/Raghav-byte.md (hunk 1 above)
Comment (Raghav-byte) [reply _data/Raghav-byte/Raghav-byte.md:5]: i did . Thanks

**Comment by mitulsavani** [comment]
Hey, you clearly haven't solved this. Her's how [file](https://raw.githubusercontent.com/Raghav-byte/graduation/33e6de1c6972b4fad0527c89ff6937fcdc2721c4/_data/Raghav-byte/Raghav-byte.md) for your reference.

**Pull request closed without merging**